### PR TITLE
Docs update #1

### PR DESCRIPTION
**Pull request by smittytone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,82 +17,100 @@ This version of the library supports the following functionality:
...
 
-More about authentication [here](https://developer.amazon.com/docs/dash/lwa-web-api.html) and [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html).
+You can read more about authentication [here](https://developer.amazon.com/docs/dash/lwa-web-api.html) and [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html).
 
-**Note**: After each restart of the agent the `Refresh Token` should be passed in to the library. So if you don't want to go through the authentication steps again, you may save the token in the agent's persistent storage and set it with the [setRefreshToken()](#setrefreshtokenrefreshtoken) method after each restart of the agent. See [the provided example](#example-store-and-load-refresh-token).
+**Note** The Refresh Token must be passed in to the library every time the agent restarta. If you don't want to go through the above authentication steps again, save the Token in the agent's persistent storage and set it with [*setRefreshToken()*](#setrefreshtokenrefreshtoken) after each agent restart. Please see [the provided example](#examples) for more details.
```
Comment (ragrus-nbl) [inline README.md]: `restarta` - a typo

File: README.md [hunk 2]
```diff
@@ -17,82 +17,100 @@ This version of the library supports the following functionality:
...
-Due to this the [login()](#loginrocky-devicemodel-deviceserial-onauthenticated-route-testdevice) method has a parameter *testDevice*. But if you set a `Refresh Token` manually with the [setRefreshToken()](#setrefreshtokenrefreshtoken) method, only you know whether this token was obtained for testing or not and such *testDevice* parameter is not required here.
+For testing purposes, Amazon DRS allows you to submit [test orders](https://developer.amazon.com/docs/dash/test-device-purchases.html). Test orders are those made by a DRS device authenticated as a test device.
 
-Only test orders can be canceled with the [cancelTestOrder()](#canceltestorderslotid-oncanceled) method.
+As such, [*login()*](#loginrocky-devicemodel-deviceserial-onauthenticated-route-testdevice) has a parameter, *testDevice*, which takes a test device ID. However, if you set a Refresh Token manually with [*setRefreshToken()*](#setrefreshtokenrefreshtoken), only you know whether this token was obtained for testing or not and so *testDevice* is not required in this case.
```
Comment (ragrus-nbl) [inline README.md]: *testDevice* parameter takes a boolean value, not ID

File: README.md [hunk 3]
```diff
@@ -17,82 +17,100 @@ This version of the library supports the following functionality:
...
+#### onAuthenticated Callback Parameters ####
 
-| Parameter | Data Type | Description |
+| Parameter | Type | Description |
 | --- | --- | --- |
-| *error* | Integer | `0` if the authentication is successful, an [error code](#error-code) otherwise. See possible HTTP error codes [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#access-token-errors). |
-| *response* | Table | Key-value table with the response provided by Amazon server. May be `null`. [See here about the response format](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#access-token-response). Also may contain error details described [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#access-token-errors) and [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#authorization-errors). |
+| *error* | Integer | `0` if the authentication is successful, otherwise an [error code](#error-code) otherwise. Possible HTTP error codes [are listed here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#access-token-errors) |
```
Comment (ragrus-nbl) [inline README.md]: `otherwise` written twice. I suppose it is a mistake

File: README.md [hunk 4]
```diff
@@ -144,37 +167,46 @@ client.setRefreshToken(AMAZON_DRS_REFRESH_TOKEN);
...
+| *onReplenished* | Function | Optional | Callback function called when the operation is completed or an error occurs |
 
-#### Callback: onReplenished(*error, response*) ####
+#### onReplenished Callback Parameters ####
 
 | Parameter | Data Type | Description |
 | --- | --- | --- |
-| *error* | Integer | `0` if the operation is completed successfully, an [error code](#error-code) otherwise. See possible HTTP error codes [here](https://developer.amazon.com/docs/dash/replenish-endpoint.html#error-responses). |
-| *response* | Table | Key-value table with the response provided by Amazon server. May be `null`. [See response example](https://developer.amazon.com/docs/dash/replenish-endpoint.html#response-example). Also may contain [error details](https://developer.amazon.com/docs/dash/replenish-endpoint.html#error-responses). |
+| *error* | Integer | `0` if the authentication is successful, otherwise an [error code](#error-code) otherwise. Possible HTTP error codes [are listed here](https://developer.amazon.com/docs/dash/replenish-endpoint.html#error-responses) |
```
Comment (ragrus-nbl) [inline README.md]: `authentication` - I suppose it is a copy-paste mistake. And `otherwise` written twice as above.

File: README.md [hunk 5]
```diff
@@ -185,33 +217,38 @@ client.replenish(AMAZON_DRS_SLOT_ID, onReplenished.bindenv(this));
...
+| *onCanceled* | Function | Optional | Callback function called when the operation is completed or an error occurs |
 
-#### Callback: onCanceled(*error, response*) ####
+#### onCanceled Callback Parameters ####
 
 | Parameter | Data Type | Description |
 | --- | --- | --- |
-| *error* | Integer | `0` if the operation is completed successfully, an [error code](#error-code) otherwise. See possible HTTP error codes [here](https://developer.amazon.com/docs/dash/canceltestorder-endpoint.html#error-responses). |
-| *response* | Table | Key-value table with the response provided by Amazon server. May be `null`. [See response example](https://developer.amazon.com/docs/dash/canceltestorder-endpoint.html#response-example). Also may contain [error details](https://developer.amazon.com/docs/dash/canceltestorder-endpoint.html#error-responses). |
+| *error* | Integer | `0` if the authentication is successful, otherwise an [error code](#error-code) otherwise. Possible HTTP error codes [are listed here](https://developer.amazon.com/docs/dash/canceltestorder-endpoint.html#error-responses) |
```
Comment (ragrus-nbl) [inline README.md]: Same as above

File: README.md (hunk 1 above)
Comment (smittytone) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (smittytone) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (smittytone) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (smittytone) [reply README.md]: Fixed

**Pull request merged**